src/microbe_stage/editor/PatchMapNode.cs

```diff
@@ -96,11 +123,23 @@ public override void _GuiInput(InputEvent @event)
                 Pressed: true, ButtonIndex: (int)ButtonList.Left or (int)ButtonList.Right,
             })
         {
+            ((PatchMapDrawer)GetParent()).MarkDirty();
             OnSelect();
             GetTree().SetInputAsHandled();
         }
     }
 
+    public override void _Process(float delta)
+    {
+        currentFadeTime += delta;
```

Comment:
Does this do the blinking of the currently selected patch? If so these related variables are badly named. Also put a comment explaining what these timer lines do, or put in to a separate method with a descriptive name.